src/richie/apps/core/templatetags/rfc_5646_locale.py

```diff
@@ -1,28 +1,31 @@
 """Custom template tags for the search application of Richie."""
 from django import template
 from django.core.exceptions import ImproperlyConfigured
-from django.utils.translation import to_locale
 
-from ..defaults import REACT_LOCALES
+from ..defaults import RFC_5646_LOCALES
 
 # pylint: disable=invalid-name
 register = template.Library()
 
 
 @register.filter
-def react_locale(value):
+def rfc_5646_locale(value):
     """
-    Converts a language (simple ISO639-1 or full language with regions) to an ISO15897 locale
-    that is supported by the React frontend.
+    Converts a language (simple ISO639-1 or full language with regions) to a BCP47/RFC5646 locale.
+    This will allow us to use this locale on the <html lang> attribute directly.
     """
-    value_locale = to_locale(value)
+    p = value.find("_")
+    if p >= 0:
```

Comment:
could be written as:
```python
if "_" in value:
```

Reply:
we're reusing this index 2 times in the line right below. IMO it's ok like this?